apps/client/pages/404.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import type { Story, Meta } from '@storybook/react'
+import Page404 from './404'
+
+export default {
+    title: 'pages/Page404',
+    component: Page404,
+} as Meta
+
+const Template: Story = () => <Page404 />
```

Comment:
does doing a named export here work as well? I think we can just do export const .... and skip the next line
still learning Storybook so apologies if this is a dumb question

Reply:
> does doing a named export here work as well? I think we can just do export const .... and skip the next line 

Yes that's a valid approach, and is more suitable in the context of this page. The `bind({})` part is used to create a new instance of the function with its own context, but it’s more relevant in scenarios where you need to create multiple stories from the same template with different properties. 

For a simple single story like this for the 404 page directly exporting as a named export is fine. 

Instead of: 
```jsx
const Template: Story = () => <Page404 />;
export const Default = Template.bind({});
``` 
We would instead change it to:
```jsx
export const Default: Story = () => <Page404 />;
```

Thanks for the suggestion.

Reply:
Interesting, learned something new!